overlay-debs/linux-rb1/linux_6.15.3-1qcom1.debdiff

```diff
@@ -0,0 +1,137 @@
+diff -Nru linux-6.15.3/debian/changelog linux-6.15.3/debian/changelog
+--- linux-6.15.3/debian/changelog	2025-06-19 22:00:04.000000000 +0000
++++ linux-6.15.3/debian/changelog	2025-06-23 14:29:32.000000000 +0000
+@@ -1,3 +1,17 @@
++linux (6.15.3-1qcom1) trixie; urgency=medium
++
++  * Set CONFIG_PINCTRL_SM8150=y, CONFIG_INTERCONNECT_QCOM_SM8150=y.
++  * Set CONFIG_CLK_QCM2290_GPUCC=m, CONFIG_RPMSG_CHAR=m, CONFIG_RPMSG_CTRL=m,
++    CONFIG_RPMSG_VIRTIO=y.
++  * Set CONFIG_QCOM_TZMEM_MODE_SHMBRIDGE=y, CONFIG_REGULATOR_QCOM_REFGEN=m.
++  * Set CONFIG_PINCTRL_QCM2290=y, CONFIG_QCM_GCC_2290=y,
++    CONFIG_QCOM_CLK_SMD_RPM=y, CONFIG_QCOM_APCS_IPC=y.
++  * Set CONFIG_CLK_QCM2290_GPUCC=y, CONFIG_RPMSG_QCOM_SMD=y,
++    CONFIG_QCOM_SMEM=y, CONFIG_QCOM_SMD_RPM=y,
++    CONFIG_INTERCONNECT_QCOM_SMD_RPM=y
++
++ -- Loic Minier <loic.minier@oss.qualcomm.com>  Mon, 23 Jun 2025 14:29:32 +0000
++
+ linux (6.15.3-1~exp1) experimental; urgency=medium
+ 
+   * New upstream stable update:
+diff -Nru linux-6.15.3/debian/config/arm64/config linux-6.15.3/debian/config/arm64/config
+--- linux-6.15.3/debian/config/arm64/config	2025-06-19 16:10:05.000000000 +0000
++++ linux-6.15.3/debian/config/arm64/config	2025-06-23 14:29:32.000000000 +0000
+@@ -187,6 +187,7 @@
+ CONFIG_COMMON_CLK_SCPI=m
+ CONFIG_COMMON_CLK_SI5341=y
+ CONFIG_COMMON_CLK_FSL_SAI=y
++CONFIG_CLK_QCM2290_GPUCC=y
+ CONFIG_CLK_QORIQ=y
+ CONFIG_CLK_LS1028A_PLLDIG=y
+ CONFIG_COMMON_CLK_XGENE=y
+@@ -364,7 +365,7 @@
+ CONFIG_QCOM_CLK_APCS_MSM8916=m
+ CONFIG_QCOM_CLK_APCC_MSM8996=m
+ CONFIG_QCOM_CLK_RPM=m
+-CONFIG_QCOM_CLK_SMD_RPM=m
++CONFIG_QCOM_CLK_SMD_RPM=y
+ CONFIG_QCOM_CLK_RPMH=y
+ CONFIG_MSM_GCC_8916=y
+ CONFIG_MSM_GCC_8996=y
+@@ -372,7 +373,7 @@
+ CONFIG_MSM_GCC_8998=y
+ CONFIG_MSM_GPUCC_8998=m
+ CONFIG_MSM_MMCC_8998=m
+-CONFIG_QCM_GCC_2290=m
++CONFIG_QCM_GCC_2290=y
+ CONFIG_QCM_DISPCC_2290=m
+ CONFIG_SC_CAMCC_7180=m
+ CONFIG_SC_CAMCC_7280=m
+@@ -1031,7 +1032,9 @@
+ CONFIG_INTERCONNECT_QCOM_SC8280XP=m
+ CONFIG_INTERCONNECT_QCOM_SDM845=y
+ CONFIG_INTERCONNECT_QCOM_SM6115=y
++CONFIG_INTERCONNECT_QCOM_SM8150=y
```

Comment:
Only for SM8150-based boards (we do not support any)

Reply:
dropped, thanks